### DISCUSSION TITLE
https://delphinqa.ling.washington.edu/ unstable

### DISCUSSION THREAD

**Discussion by arademaker** [delph-in/docs#139] [Announcements]
Should we start to rethink alternatives? Zulip, Discord, GitHub... Surely we need to consider the pros and cons and the fact that we are a small community that wants to preserve as much as possible our data and conversations.

**Comment by emilymbender** [comment]
Some key desiderata:

1) Data portability. If we move away from the delphinqa discourse server, we'd want to bring the discussions from there to the new site.
2) UI -- the site should be set up for discussion, rather than be a discussion add-on to a site meant for something else. In other words. PLEASE NOT GITHUB.

**Reply by arademaker** [reply]
I agree with both @emilymbender! GitHub is not the perfect platform; this forum is very limited. Take a look at the https://leanprover.zulipchat.com community; very active and seems to be a flexible tool.  I just sent a message to Zulip support about it. We also have Slack and many others. But I agree 100% with your concern about data portability.

**Comment by olzama** [comment]
I believe the main reason it is unstable is that the admin resources UW can spare are not always sufficient. In other words, the problem is not with the discourse forum technology itself. Rather, it is that it is not easy for the UW admin who has many other duties to set it up correctly and then maintain it properly at all times... The exact same will be true with any other technology, if it is set up locally by any institution.

That being said, I believe it has been quite stable for the last 2-3 years, until a few months ago when email stopped working. It is possible that after (if) that gets fixed, it will become stable again.

**Reply by arademaker** [reply]
The main issue is not the technology but the IT service to admin it. We could move the forum to a cloud hosting solution with a free plan for a small community. We want to (1) delegate the admin for a cloud instance; (2) don't have costs with that given the size of our community. In https://zulip.com/plans/, the free option should work for us... Other platforms may also have similar hosting plans. 

https://www.discourse.org/pricing doesn't have a free plan. Probably UW bought a license some time ago.
